packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/configmid.md

```diff
@@ -0,0 +1,24 @@
+The Okta Angular SDK requires an instance of an `OktaAuth` object with configuration properties. You should set the `ClientID` and `Issuer` properties. This can happen by directly setting the properties, with variable replacement that happens as part of the build process, or during application load time.
+
+Make the following changes to `src/app/app.module.ts`:
+
+1. Add the following import lines to the code to pull in the dependencies:
+
+```ts
+import { OktaAuthModule, OKTA_CONFIG } from '@okta/okta-angular';
+import { OktaAuth } from '@okta/okta-auth-js';
+```
+
+2. Add `OktaAuthModule` to the `@NgModule` `imports` array.
+
+3. Add a new `OktaAuth` object like the one below, replacing the `issuer` and `clientId` values with your own values:
+
+```ts
+const oktaAuth = new OktaAuth({
+  issuer: 'https://{yourOktaDomain}/oauth2/default',
```

Comment:
I've noticed that some guides have `${yourOktaDomain}`, while others don't have the dollar sign prefix. On the developer blog, we don't use a dollar sign.

Reply:
I'm going to change all these to use the dollar sign, as we tend to use it in the dev.okta content.